### PR TITLE
Remove Problematic PDFs

### PR DESCRIPTION
**Pull request by jlevypaloalto**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [Remove Usage of Problematic Test Data](https://jira-hq.paloaltonetworks.local/browse/CIAC-7097)

## Description
Removes test PDFs that are copyright and/or Intellectual property.

## Screen Shots
**The new PDFs:**
Screenshot 2023-07-09 at 13 06 54
Screenshot 2023-07-10 at 17 35 20
Screenshot 2023-07-10 at 18 06 22

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by JudahSchwartz** [comment]
check this test is still working Packs/CommonScripts/TestPlaybooks/playbook-ReadPDFFileV2_Test.yml

**Review by JudahSchwartz — COMMENTED** [review]
is this ready for review?

**Comment by jlevypaloalto** [comment]
I found some more PDFs to remove, I've made a list in the [ticket](https://jira-hq.paloaltonetworks.local/browse/CIAC-7097).

File: Packs/CommonScripts/Scripts/ReadPDFFileV2/ReadPDFFileV2_test.py [hunk 1]
```diff
@@ -195,7 +195,7 @@
         assert 'Incorrect password' in str(e)
     # to_html_output_folder = f'{tmp_path}/PDF_html'
     html_text = get_pdf_htmls_content(f'{CWD}/hyperlinks.pdf', tmp_path)
-    assert 'http://www.antennahouse.com/purchase.htm' in html_text
+    assert 'http://www.example.com/' in html_text
```
[the remark this answers is not in the document]
Comment (jlevypaloalto) [reply Packs/CommonScripts/Scripts/ReadPDFFileV2/ReadPDFFileV2_test.py:198]: Dismissed

**Review by JudahSchwartz — APPROVED** [review]
Great!! thanks!

**Pull request merged**